model/command_response.go

```diff
@@ -32,31 +33,30 @@ type CommandResponse struct {
 }
 
 func CommandResponseFromHTTPBody(contentType string, body io.Reader) (*CommandResponse, error) {
-	if strings.TrimSpace(strings.Split(contentType, ";")[0]) == "application/json" {
-		return CommandResponseFromJSON(body)
-	}
-	if b, err := io.ReadAll(body); err == nil {
-		return CommandResponseFromPlainText(string(b)), nil
+	if bodyRaw, err := io.ReadAll(body); err != nil {
+		return nil, nil
```

Comment:
Should we return this error here?

Reply:
Original implementation was shadowing this error with 'Nil'. I've left this as it was